examples/apps/generic_client/generic_client.py

```diff
@@ -47,14 +47,17 @@ def ParseCommandLine(args) :
 	global mode
 	global uri
 	global address
+	global show_receipt
+	global show_decrypted_output
```

Comment:
Number of global variables ins increasing :-). We could probably raise another PR to reorganize things a bit.